### ISSUE TITLE
Does this component also scan bar code?

### ISSUE THREAD

**Issue by arthurmmedeiros** [JodusNodus/react-qr-reader#152]
I'm just trying to figure if this component does both types of scanning automatically

**Comment by JosNun** [comment]
No, I had to implement a separate barcode scanner. I ended up making a wrapper around [Quagga.js](https://github.com/serratus/quaggaJS)

**Comment by arthurmmedeiros** [comment]
Nice, is there any plan to make Quagga read Qr code?

**Comment by JosNun** [comment]
That's probably a better question to ask the Quagga.js folks
